src/terra-dev-site/doc/About.md

```diff
@@ -0,0 +1,274 @@
+# Terra Application
+
+The `terra-application` package contains the components that comprise Terra's application platform.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
```

Comment:
For this package, I think it would be important to list what terra-application provides as dependencies so a project/library doesn't provide it's own versions

Reply:
Anything that we're providing that we don't want duplicated should be caught by the duplicate dependency plugin. I'd rather have that act as living safeguard vs. having to maintain a list in markdown.